### DISCUSSION TITLE
iTwin exporter for datasmith silently crashes while exporting

### DISCUSSION THREAD

**Discussion by Leave1223** [iTwin/itwinjs-core#3976] [General] [answered]
When exporting the itwin for datamith for unreal, the window closes and doenst give , the export .uadata

**Comment by mgooding** [comment]
The developer for this product is unavailable today, but he suggested that he'll most likely need access to your model to debug this issue. Can you email mike.belousov@bentley.com to follow up and work through this issue?

**Comment by MichaelBelousov** [comment]
hi @Leave1223 it sounds like the application is crashing. Could you reach out to me with the lastest debug log after the crash? (see the help menu in the top right).

**Reply by arnab-git-hub** [reply]
@Leave1223 Hi, it seems you have not yet provided the debug logs. Can you please provide the same by emailing me ([arnab.gogoi@bentley.com](arnab.gogoi@bentley.com)) or [mike.belousov@bentley.com](mailto:mike.belousov@bentley.com). If you need help in extracting the crash logs, let us know.

**Comment by Leave1223** [comment, marked as answer]
Dear Team,It has actually started working now 


-------- Original Message --------
Subject: Re: [iTwin/itwinjs-core] ITWIN exporter for datasmith not
working (Discussion #3976)
From: Arnab Kumar Gogoi ***@***.***>
Date: Wed, August 03, 2022 4:35 pm
To: iTwin/itwinjs-core ***@***.***>
Cc: Leave1223 ***@***.***>,  Mention ***@***.***>

 @Leave1223 Hi, it seems you have not yet provided the debug logs. Can you please provide the same by emailing me ***@***.***) or ***@***.*** If you need help in extracting the crash logs, let us know. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Reply by MichaelBelousov** [reply]
Great! Please open another discussion or issue if you have problems again.
